.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Regarding all those files:
Place them in `.gitignore` (can use the root one) and then you can run:
```shell
git rm [-r] --cached {file path}
```

to remove them from git